src/app/pages/registration/register-user.jsx

```diff
@@ -109,67 +131,64 @@ function ChplRegisterUser({ hash }) {
       case 'login':
         return (
           <>
-            <Typography>{ message }</Typography>
             <ChplCognitoLogin
               dispatch={handleDispatch}
               state={cognitoLoginComponentState}
               setState={setCognitoLoginComponentState}
             />
-            <Typography>
-              Or
-              {' '}
+            <Box className={classes.cardFooter}>
+              <Typography variant="body2">
+                Don&apos;t have an account?
+              </Typography>
               <Button
                 color="primary"
                 variant="outlined"
-                onClick={() => setState('create')}
+                size="small"
+                onClick={() => setState('create')} // Switch to login state
               >
-                create a new account
+                Create a new account
               </Button>
-            </Typography>
+            </Box>
           </>
         );
       case 'create':
         return (
           <>
-            { message.length > 0
-              && (
-                <Typography
-                  color="error"
-                >
-                  { message }
-                </Typography>
-              )}
             <ChplCognitoUserCreate dispatch={handleDispatch} />
-            <Typography>
-              Or
-              {' '}
+            <Box className={classes.cardFooter}>
+              <Typography>
+                Have an account?
+              </Typography>
               <Button
                 color="primary"
                 variant="outlined"
-                onClick={() => setState('cognito-login')}
+                size="small"
+                onClick={() => setState('login')} // Switch to login state
               >
-                log in to your existing account
+                Log in to your existing account
               </Button>
-            </Typography>
+            </Box>
           </>
         );
-      case 'success':
```

Comment:
Is it impossible to get into this state? If it's possible, then we should have something here